rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -84,8 +84,32 @@ def __init__(
     def get_positions(
             self,
             given_addresses: Sequence[ChecksumEvmAddress],
-    ) -> dict[ChecksumEvmAddress, Trove]:
-        """Query liquity contract to detect open troves"""
+    ) -> dict[str, Any]:
```

Comment:
Can you maybe use a namedtuple here or a typed dict (https://peps.python.org/pep-0589/) This will help mypy and prevent errors for example in the aggregator

Reply:
for the response object right?
have added one for the same